src/coffee/client.coffee

```diff
@@ -220,9 +226,11 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    if options.enableRepeater? and !options.enableRepeater
```

Comment:
Hm. True. Then I think this is not the right place to have this default fallback. It should be done like [here](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/client.coffee#L228). But this is a general problem in the SDK right now as far as I can see.